refine-nextjs/plugins/inferencer-headless/pages/categories/edit/[id].tsx

```diff
@@ -21,6 +21,15 @@ export default function CategoryEdit() {
     <%_ if (answers["data-provider"] === 'data-provider-hasura') { _%>
     meta={inferencerPredefinedMeta}
     <%_ } _%>
+    <%_ if (answers["data-provider"] === 'data-provider-strapi-v4') { _%>
+    fieldTransformer={(field: any) => {
```

Comment:
```suggestion
    fieldTransformer={(field: any) => {
```

Can we remove `any` here? Are there any errors here if we don't type it `any`?

Reply:
Will be addressed in the next PR.